Scheduler/feature_scheduler/maintel/fbs_config_minimal_fieldsurvey.py

```diff
@@ -67,60 +88,31 @@ def get_scheduler():
             max_alt=83.0,
             shadow_minutes=2.0,
         ),
-        basis_functions.SlewtimeBasisFunction(filtername="u_02", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="g_01", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="r_03", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="i_06", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="z_03", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="y_04", nside=nside),
-        basis_functions.FilterLoadedBasisFunction(filternames=sequence),
+        basis_functions.SlewtimeBasisFunction(bandname="u", nside=nside),
+        basis_functions.SlewtimeBasisFunction(bandname="g", nside=nside),
+        basis_functions.SlewtimeBasisFunction(bandname="r", nside=nside),
+        basis_functions.SlewtimeBasisFunction(bandname="i", nside=nside),
+        basis_functions.SlewtimeBasisFunction(bandname="z", nside=nside),
+        basis_functions.SlewtimeBasisFunction(bandname="y", nside=nside),
+        # TODO: check if BandLoadedBasisFunction is needed
+        basis_functions.BandLoadedBasisFunction(bandnames=sequence),
```

Comment:
You should NOT use this basis function if you want the sequence to execute even if one of the bands is unavailable. If you want the sequence to fail is one of the bands is unavailable, leave it in.